proprietary/tokens/src/components/ams/header.tokens.json

```diff
@@ -7,6 +7,12 @@
         "value": "{ams.grid.padding-inline}",
```

Comment:
Can you move this file to `packages-proprietary`?

Reply:
Yes, I repaired the merge.